### ISSUE TITLE
Questions on switching other models with your sampling method

### ISSUE THREAD

**Issue by TianwenZhou** [yuanzhi-zhu/DiffPIR#26]
Dear Authors,
Thank you so much for your great work, as far as I am concerned, you only tried two types of pre-trained diffusion models in your work. Could you please shed some lights on ways to use other pretrained diffusion models (SD-v2.1 etc.) with your sampling method?
Thank you again!

**Comment by yuanzhi-zhu** [comment]
Hi @TianwenZhou ,

Thank you for your interest. Actually, most of the previous work including DiffPIR, DPS, and DDRM, rely on diffusion UNets in image space (as image denoisers).

To release the generative power of large diffusion models such as SD, I recommend you have a look at https://arxiv.org/abs/2310.01110 & https://arxiv.org/abs/2311.15658 from @HJ-harry

**Comment by TianwenZhou** [comment]
Thank you for your letter, I will reply ASAP.BWTianwen Zhou

**Comment by TianwenZhou** [comment]
Dear @yuanzhi-zhu ,
Thank you so much for your prompt reply, it is really helpful for me! As I was browsing through the pretrained diffusion models, I only found unconditional model trained on 256x256 ImageNet, while models with other resolutions(e.g. 512x512) are conditional. Given the fact that training such a huge model from scratch is unrealistic, I wonder if you could kindly offer the method to load pretrained conditional 512x512 diffusion model to your sampling process.

**Comment by yuanzhi-zhu** [comment]
unfortunately, to the best of my knowledge, there is no 512*512 **image space** diffusion model open-sourced.

**Comment by TianwenZhou** [comment]
So are there any ways that I can use a class-conditional 512x512 image space diffusion [model](https://openaipublic.blob.core.windows.net/diffusion/jul-2021/512x512_classifier.pt) with your sampling strategy?

**Comment by yuanzhi-zhu** [comment]
sure, assume that you know the class of the corrupted image, you can use the guided score as $p(x)$.

**Comment by TianwenZhou** [comment]
Thank you, I'm also curious about ways to test directly on low resolution images, rather than degrading the HR image following the given degradation process

**Comment by TianwenZhou** [comment]
Thank you so much for your kind help and guidance! I'll definitely test these out

**Comment by peertechz** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
RustMan88, @guiluchesi, @lisovskyvlad, @coyg, @yqrm, @zhamppx97, @matthieucan, @shahnawaz6alam, @joseph-hurtado, @nestorPons, @ferasassaf, @kushagra765, @Revlin, @guoxiaoqi, @ProgramForFun, @cjmdaixi, @uldissturms, @alexander480, @atmshop, @wingyiu

**Comment by GONGJIA0208** [comment]
@TianwenZhou  Hi, tianwenZhou, have you solved this problem? I also want to try to use SD1.5 for inpainting task

**Comment by TianwenZhou** [comment]
@GONGJIA0208 Hi, are you willing to try conditional diffusion models with other resolutions?
